man/sd_bus_message_append.xml

```diff
@@ -109,6 +118,13 @@
     values for each entry matching the element type of
     the dictionary entries.</para>
 
+
+    <para>The <function>sd_bus_message_appendv()</function> is equivalent to
+    the function <function>sd_bus_message_append()</function>, except that
+    it is called with a va_list instead of a variable number of arguments.
+    This function does not call the va_end macro. Because it invokes the
+    va_arg macro, the value of ap is undefined after the call.</para>
```

Comment:
va_end and va_arg should probably be suffixed with "()", as it is typical in the documentations of our C functions. Yes, strictly speaking both are macros, and not functions, but they definetely have "function-like" usage, and hence I'd still place the "()" suffix on both. In addition I'd suggest to enclose each in `<function>`.